peer/peer.go

```diff
@@ -1867,6 +1886,8 @@ func (p *Peer) readRemoteVersionMsg() error {
 		errStr := "A version message must precede all others"
 		log.Errorf(errStr)
 
+		p.AddBanScore(1, 0, errStr)
```

Comment:
Why not AddBanScoreAndPushRejectMsg, since you are pushing Reject in next line?

Reply:
Because you don't have a output handler in this phase